tests/integration/solidity_contracts/PlainOpcodes/PlainOpcodes.sol

```diff
@@ -127,4 +128,52 @@ contract PlainOpcodes {
             extcodecopy(target, add(extcode, 0x20), offset, size)
         }
     }
+
```

Comment:
Currently putting exception handling tests in plainopcodes. I think it makes sense to refactor into a whole ExceptionHandling directory in integration testing. Thoughts?